server/paymentProviders/stripe/creditcard.js

```diff
@@ -272,6 +274,8 @@ export default {
       if (error.message !== 'Payment Intent require action') {
         logger.error(`Stripe Payment Error: ${error.message}`);
         logger.error(error);
+        logger.error(error.stack);
+        throw new Error(UNKNOWN_ERROR_MSG);
```

Comment:
This will send a generic error message to users asking them to contact support. The rationale is that stripe error message can contain sensitive information and that it's often cryptic to the user.

We should iterate by whitelisting or translating errors that we know are interesting to display for the user.